website/src/css/pages/_jobs.scss

```diff
@@ -245,6 +245,26 @@
         }
     }
 
+    .reviews-carousel {
+        height: 800px;
+        display: table;
+        width: 100%;
+        @media (max-width: 768px) {
+            height: 775px;
+        }
+        @media (max-width: 576px) {
+            height: 650px;
+        }
+
+        .carousel-inner {
+            display: table-cell;
```

Comment:
`d-table-cell` in Bootstrap

Reply:
`.carousel-inner` is another tag added by Bootstrap in between `Carousel` and `Carousel.Item` though 🤔  , and doesn't seem I can get to it in the template by something like `Carousel.Inner`. Any idea how I can add this utility class inline to this hidden element from the template?
![image](https://user-images.githubusercontent.com/59381432/161163558-e8c0037e-aece-41fa-ba86-a54f59ab205a.png)

Reply:
Disregard, obvious solution of adding an extra wrapper to the CarouselItem did it 🤦‍♀️  -- Custom styles here are nixed!

Reply:
Awesome! Thanks for nuking the styles 🎉